brave/brave-core
# Restore browser window before showing Rewards panel when solving adaptive captcha

Resolves https://github.com/brave/brave-browser/issues/18858

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Clean profile
- Enable Rewards
- Schedule an adaptive captcha for your wallet id
- Minimize browser
- Click "Solve" when captcha notification window appears
- Ensure that browser window is restored and Rewards panel with captcha is displayed correctly
